MaraBot/Messages/Display.cs

```diff
@@ -115,13 +117,43 @@ private static DiscordEmbedBuilder AddOptionsToEmbed(this DiscordEmbedBuilder em
                 }
             }
 
-            embed.AddField("Options", optionStrings[0], true);
+			embed.AddField(title, optionStrings[0], optionStrings.Length > 1);
             for (int i = 1; i < optionStrings.Length; ++i)
-            {
                 embed.AddField("\u200B", optionStrings[i], true);
-            }
+
+			return embed;
+		}
+
+        private static DiscordEmbedBuilder AddOptionsToEmbed(this DiscordEmbedBuilder embed, Preset preset)
+        {
+			Dictionary<string, string> generalOptions = new Dictionary<string, string>();
+			Dictionary<string, string> modeOptions = new Dictionary<string, string>();
+			Dictionary<string, string> otherOptions = new Dictionary<string, string>();
+
+			Mode mode = Option.StringToMode(preset.Options["mode"]);
+			var options = preset.MakeDisplayable();
```

Comment:
I don't think we should do this work when we display the options, but rather when we build the preset list when first initializing the bot.

The idea being that when queried for a preset, race, weekly, the bot should not have to actively translate the options, but they should be already translated.

Perhaps we should have the three dictionaries in the Preset class alongside the raw options.

Something like:

```
class Preset
{
    [JsonPropertyName("Options")] public Dictionary<string, string> RawOptions;
    [JsonIgnoreAttribute] public Dictionary<string, string> GeneralOptions;
    [JsonIgnoreAttribute] public Dictionary<string, string> ModeOptions;
    [JsonIgnoreAttribute] public Dictionary<string, string> OtherOptions;

    ...
}
```